v22.1/crdb-internal.md

```diff
@@ -578,6 +578,156 @@ WHERE metadata @> '{"db":"movr"}' AND (metadata @> '{"stmtTyp":"TypeDDL"}' OR me
 
 ~~~
 
+### `transaction_contention_events`
+
+Requires either the `VIEWACTIVITY` or `VIEWACTIVITYREDACTED` [role option](alter-role.html#role-options) to access. If you have the `VIEWACTIVITYREDACTED` role, `contending_key` will be redacted.
+
+Contention events are stored in memory. You can control the amount of contention events stored per node via the `sql.contention.event_store.capacity` [cluster setting](cluster-settings.html).
```

Comment:
Do we expect users to configure this cluster setting? I'd remove this for now and prefer not to provide guidance and treat it similar to `sql.contention.event_store.duration_threshold` where the setting is only an escape hatch if we run into issues with this feature. 

@Azhng, do you know if there's a more logical/user-friendly limitation we impose on this table like a Max row limit (versus size)? Like what we have for persisted stats of 1M rows.

Reply:
Since this `transaction_contention_events` table is an in-memory only table, applying a global cluster-level size limit is quite difficult. I think one reasonable global cluster-level limit we can introduce would be an event TTL. Though even with a global TTL limit, we would still want a per-node size limit like what we have today.